controller/user-controller.js

```diff
@@ -0,0 +1,32 @@
+const User = require("../models/user");
+
+exports.getCurrentUser = async (req, res) => {
```

Comment:
Can you explain what exactly you want from this current user?

Reply:
the user who is logged in currently..